example/clone-recipe.json

```diff
@@ -9,7 +9,7 @@
     "sandboxServerInfo": {
       "secure": false,
       "port": 9550,
-      "host": "127.0.0.1"
+      "host": "0.0.0.0"
```

Comment:
This would expose the connector to the outside network. This is something that should be opted in to not enabled by default.